besu/src/main/java/org/hyperledger/besu/controller/CliqueBesuControllerBuilder.java

```diff
@@ -46,6 +47,7 @@
 import org.apache.logging.log4j.Logger;
 
 public class CliqueBesuControllerBuilder extends BesuControllerBuilder<CliqueContext> {
```

Comment:
Is this PR for Clique as well as IBFT?

Reply:
Ah - had to include this here as the Metrics plugin work was affected by the change